lms/static/certificates/sass/_components.scss

```diff
@@ -569,3 +572,89 @@
     }
 }
 
+// ------------------------------
+// #BADGES MODAL
+// ------------------------------
+.badges-overlay {
+
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 10;
+    background-color: rgba(0, 0, 0, 0.5); /*dim the background*/
+    vertical-align: middle;
+
+    .badges-modal {
+        color: #9b9b9b;
+        font-size: 14pt;
+        box-sizing: content-box;
+        right: 0;
```

Comment:
NIT: make sure this is [RTL syntax](https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#right-to-left-rtl-support) friendly.